src/avatar.scss

```diff
@@ -49,6 +48,7 @@ $block: #{$fd-namespace}-avatar;
       background-size: cover;
       background-position: 50%;
       border-radius: var(--sapElement_BorderCornerRadius);
+      box-sizing: border-box;
```

Comment:
Why did you add this? It's coming with the reset function.
<img width="498" alt="Screen Shot 2020-08-19 at 10 34 09 AM" src="https://user-images.githubusercontent.com/39598672/90648567-b191eb00-e207-11ea-9c7c-db802e561b8e.png">

Reply:
thanks! I removed it